scss/components/_case-custom-fields.scss

```diff
@@ -0,0 +1,88 @@
+.civicase__case-custom-fields__container {
+  float: left;
+  width: 50%;
+
+  &:first-child {
+    padding: 0 15px 24px 24px;
+  }
+
+  &:nth-child(2) {
+    padding: 0 24px 24px 15px;
+  }
+
+  .crm-editable-enabled:not(.crm-editable-editing) {
```

Comment:
Shouldn't these styles be a part of generic styles.
```
.civicase__inline-editable {
  .crm-editable-enabled:not(.crm-editable-editing) {
    &:hover {
        border: 2px dashed transparent;
        padding: 15px 20px;
      }
    }
     .crm-case-custom-form-block {
      padding: 15px 20px;
       table {
        width: 100%;
      }
    }
     .crm-submit-buttons {
      border: 0;
      padding: 15px 8px 0;
      text-align: right;
    }
     .crm-form-submit {
      min-width: auto;
      padding: 7px 19px;
       &.cancel {
        padding: 6px 19px;
         &:hover {
          color: $crm-white !important;
        }
      }
    }
  }
}
```
In this way is any other card uses inline edit for civicase these styles can be reused by adding `civicase__inline-editable` class to the wrapper HTML.

What do you think?

Reply:
To add these styles for custom fields add this class `civicase__inline-editable` to  the HTML.
```html
<div class="civicase__case-custom-fields__container civicase__inline-editable">
```

Reply:
I dont think so, because the the overrides we did is only for the Custom Fields. We dont want the samething outside of custom fields. `crm-editable-enabled` is used many places in civicase, but they dont need this customisation.